src/Offer/EditOfferRestController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace CultuurNet\UDB3\Symfony\Offer;
+
+use CultuurNet\UDB3\Offer\OfferEditingServiceInterface;
+use CultuurNet\UDB3\UsedLabelsMemory\UsedLabelsMemoryServiceInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class EditOfferRestController
+{
+    /**
+     * @var OfferEditingServiceInterface
+     */
+    private $editService;
+
+    /**
+     * @var \CultureFeed_User
+     */
+    private $user;
+
+    /**
+     * @var UsedLabelsMemoryServiceInterface
+     */
+    private $labelMemory;
+
+    /**
+     * EditOfferRestController constructor.
+     * @param OfferEditingServiceInterface $editingServiceInterface
+     */
+    public function __construct(
+        OfferEditingServiceInterface $editingServiceInterface,
+        \CultureFeed_User $user,
+        UsedLabelsMemoryServiceInterface $labelMemory
+    ) {
+        $this->editService = $editingServiceInterface;
+        $this->user = $user;
+        $this->labelMemory = $labelMemory;
+    }
+
+    /**
+     * @param Request $request
+     * @param $cdbid
+     * @return JsonResponse
+     */
+    public function addLabel(Request $request, $cdbid)
+    {
+        $response = new JsonResponse();
+
+        $label = new \CultuurNet\UDB3\Label($request->request->get('label'));
```

Comment:
Deze regel gaat denk ik niet werken in Drupal. De data wordt als JSON verstuurd in de POST request, niet als form data. Dit werkt op in Silex omdat we momenteel een middleware hebben voorzien die JSON data naar form data omzet in het Request object, maar die middleware is er niet in Drupal. We zouden hiervoor een JSONDeserializer class moeten voorzien zodat we die ook kunnen gebruiken in Drupal.